lib/TAP/Formatter/Color.pm

```diff
@@ -12,7 +12,7 @@ my $NO_COLOR;
 BEGIN {
     $NO_COLOR = 0;
 
-    eval 'use Term::ANSIColor';
+    eval 'require Term::ANSIColor';
```

Comment:
When using block eval with require, it's important to check the build output to confirm that the module did not get added to the prereq list -- it will be if you're using Dist::Zilla::Plugin::AutoPrereqs, so then corrective measures need to be taken.